### PR TITLE
fixed codacy messages, commented where unfixable

### LINKED ISSUE #369: Apply Codacy recomendation
@ngbusca added picca to a Codacy check.
Let's apply step by step the different recomendation from https://app.codacy.com/app/ngbusca/picca/issues/index

### PR DESCRIPTION
**Pull request by Waelthus**
fixes #369 for the Pk1D

### CODE REVIEW

**Comment by londumas** [comment]
Thanks @Waelthus. Let's wait for Codacy test and then merge.
Could you add me as reviewer?

File: bin/do_Pk1D.py [hunk 1]
```diff
@@ -195,8 +194,8 @@ def compute_mean_delta(ll,delta,iv,zqso):
             if (d.mean_SNR<=args.SNR_min or d.mean_reso>=args.reso_max) : continue
 
             # first pixel in forest
-            for first_pixel in range(len(d.ll)) :
-                 if (sp.power(10.,d.ll[first_pixel])>args.lambda_obs_min) : break
+            for first_pixel,first_pixel_ll in enumerate(d.ll):
+                if (sp.power(10.,first_pixel_ll)>args.lambda_obs_min) : break
```
Comment (londumas) [inline bin/do_Pk1D.py]: could you remove the parenthesis? They are useless.

Comment (Waelthus) [reply bin/do_Pk1D.py]: while doing this, does sp.power add anything vs 10**ll?

Comment (londumas) [reply bin/do_Pk1D.py]: no, `10**ll` is very good.

**Comment by ngbusca** [comment]
see https://stackoverflow.com/questions/25870923/how-to-square-or-raise-to-a-power-elementwise-a-2d-numpy-array for `power` vs `**`

**Comment by Waelthus** [comment]
I guess speed is not an issue at that position, but if the more readable power operator is faster than the power function as well, this is good.

**Comment by londumas** [comment]
@Waelthus, Thanks the PR is perfect as it is. Let's simply wait for Codacy check.

**Comment by londumas** [comment]
Thanks @Waelthus. Merged.

**Pull request merged**